src/styles/card_style.css

```diff
@@ -0,0 +1,85 @@
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.prod-img {
+  display: block;
+  width: 160px;
+  height: 134px;
+}
+
+.card__image {
+  margin: 0 auto 40px;
+}
+
+.prod-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__name {
+  margin-bottom: 4px;
+}
+
+.prod-code {
+  color: #616070;
+
+}
+
+.card__code {
+  margin-bottom: 16px;
+}
```

Comment:
you can do it in one class. better choose 'card__code'

Reply:
I agree with you, but I did that only because **"[BEM & STYLES] - Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector."**

Reply:
i see.  it is a good question for me too, but if i read strictly as they had written, .card__code is not a Block. so up to you.